rebar.config

```diff
@@ -65,11 +65,6 @@
 
 {shell, [{apps, [lager, ranch]}]}.
 
-{dialyzer, [
-    {warnings, [unknown]},
```

Comment:
I wasn't expecting you'd have to strip this section out after you fixed the abuse of handle_gossip_data?

Reply:
I had to get rid of this to get rid of the throttle errors I was seeing.  maybe I didn't need to?